### ISSUE TITLE
Comment on website

### ISSUE THREAD

**Issue by anthrolisp** [respatialized/sicp#1]
I'm not sure where to put this as I could not find a contact address or a repository for your website. I just wanted to say that I love your website, and am curious to what you'll come up with. In the meantime, I was wondering if you happen to see org-roam and package for emacs?

- https://blog.jethro.dev/posts/introducing_org_roam/
- https://github.com/jethrokuan/org-roam

Many thanks in advance!

**Comment by respatialized** [comment]
Hello - appreciate the kind words! Frankly, I'm surprised to hear that my website is discoverable at all, given that I don't post about it on any social media platforms. 

I was previously hosting my site on GitLab pages, which based on a cursory Google search, does not turn up as easily as my Github profile, so that may be why you had difficulty tracking down the website's source material! I have since moved it to Github now that the publishing options with Github Actions are flexible enough to support a custom build step. Now that my writing is hosted and published here, that repo is a more relevant place to discuss what I've written about.

https://github.com/respatialized/respatialized.github.io

Thanks for the recommendation. I'm glad people have been able to quickly develop a FOSS alternative to Roam, but I'm personally not interested in this project. I have further thoughts on why I don't feel drawn to systems like Roam or based on org-mode - I'll write them up in more detail on my website. 

*The short version*: in my experience, pretty much every knowledge management tool based on `org-mode` and emacs ties the possible outputs far too closely to the input format, and relies too heavily on the capabilities of the editor itself. This results in a relatively closed system where bridging the gap between the writing environment and other sources of information and publishing formats is difficult enough to put off forever. Extensibility is possible with elisp, of course, but I'd like any code that I write to transform or manipulate text or data to be usable _outside_ the context of my editor (e.g. in a build pipeline or even potentially on the client side).
